src/devices/CharacterLcd/IGpioController.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.CharacterLcd
+{
+    // Can we move this upstream and actually derive from this? Should I2C/SPI derive from GpioDriver instead?
```

Comment:
I would just remove this comment and instead have something like: `The work for moving this interface upstream is tracked by <issueNumber>`

Reply:
Gone